src/hotspot/os/linux/cgroupV1Subsystem_linux.cpp

```diff
@@ -76,42 +77,62 @@ void CgroupV1Controller::set_subsystem_path(char *cgroup_path) {
  */
 jlong CgroupV1MemoryController::uses_mem_hierarchy() {
   julong use_hierarchy;
-  CONTAINER_READ_NUMBER_CHECKED(this, "/memory.use_hierarchy", "Use Hierarchy", use_hierarchy);
+  CONTAINER_READ_NUMBER_CHECKED(_reader, "/memory.use_hierarchy", "Use Hierarchy", use_hierarchy);
   return (jlong)use_hierarchy;
 }
 
 void CgroupV1MemoryController::set_subsystem_path(char *cgroup_path) {
-  CgroupV1Controller::set_subsystem_path(cgroup_path);
+  _reader->set_subsystem_path(cgroup_path);
   jlong hierarchy = uses_mem_hierarchy();
   if (hierarchy > 0) {
     set_hierarchical(true);
   }
 }
 
-jlong CgroupV1Subsystem::read_memory_limit_in_bytes() {
+static inline
+void do_trace_log(julong read_mem_limit, julong host_mem) {
```

Comment:
Should it be called trace when we log on a debug level?

Reply:
renamed to `verbose_log`